frontend/package.json

```diff
@@ -3,20 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@canvasjs/react-charts": "^1.0.2",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "chart.js": "^4.4.0",
     "d3": "^7.8.5",
     "jwt-decode": "^4.0.0",
-    "path-to-regexp": ">=1.9.0",
-    "react": "^18.2.0",
-    "react-chartjs-2": "^5.2.0",
+    "path-to-regexp": "^8.2.0",
+    "performance-now": "^2.1.0",
+    "react": "^18.3.1",
+    "react-charts": "^2.0.0-beta.7",
```

Comment:
OK...so is this a completely different charting tool? Does it still work? Canvasjs was an open source chart plugin from a company, which I switched to because it had the ability to zoom in to a chart, and create step-functions. Not sure if `react-charts` has these features?